process/elasticproc/interface.go

```diff
@@ -13,6 +13,12 @@ import (
 	"github.com/multiversx/mx-chain-es-indexer-go/process/elasticproc/tokeninfo"
 )
 
+// MainChainDatabaseClientHandler defines the actions that sovereign database client handler should do
+type MainChainDatabaseClientHandler interface {
```

Comment:
Shouldn't this have an `IsInterfaceNil` func as well?

Reply:
added